projects/vdk-heartbeat/.gitlab-ci.yml

```diff
@@ -28,6 +28,28 @@ vdk-heartbeat-test:
       junit: tests.xml
 
 
+vdk-heartbeat-release-acceptance-test:
+  stage: pre_release
+  before_script:
+    - cd projects/vdk-heartbeat
+  script:
+    - set -x
+    - python setup.py sdist --formats=gztar
+    - pip install vdk-heartbeat --extra-index-url $PIP_EXTRA_INDEX_URL
+    - export VDK_HEARTBEAT_API_TOKEN=$VDK_API_TOKEN
+    - export VDK_HEARTBEAT_OP_ID="vdk-heartbeat-$CI_PIPELINE_ID"
+    - export JOB_NAME=vdk-heartbeat-release-test-job-$(date +%s)
+    - vdk-heartbeat -f cicd/cicd_heartbeat_config.ini
+  artifacts:
+    when: always
+    reports:
+      junit: tests.xml
+  only:
+    refs: # TODO switch to "main" so it is used as a pre-release test not pre-merge test
```

Comment:
It's hard to understand this TODO. Should this change trigger on merge to main, on PRs or on both?

Reply:
he has that change to test and is going to make changes before merging.